### DISCUSSION TITLE
Different number of node inputs for SplineConv

### DISCUSSION THREAD

**Discussion by hkim716** [pyg-team/pytorch_geometric#4677] [Q&A] [answered]
Hi Matt, 

I have a question about ```SplineConv``` in ```mnist_graclus.py``` example. 
For mnist superpixel dataset, the number of nodes are identical because each sample has 75 nodes.

My question is : 
If there are mixed data samples that have different ```number of nodes``` (like 60~75 nodes) and ```edge_index```, is it possible to use those samples as an input of ```SplineConv```?

**Comment by rusty1s** [comment, marked as answer]
Yes, `SplineConv` (and any other GNN layer in PyG) is agnostic to the number of nodes and supports a varying number of nodes across graphs within a mini-batch.
